js/calendar.js

```diff
@@ -1054,7 +1056,9 @@ if(!String.prototype.formatNum) {
 
 		var week = $(document.createElement('div')).attr('id', 'cal-week-box');
```

Comment:
I think this line also have to be inside conditions because this is the box itself.